weblate/utils/tasks.py

```diff
@@ -83,27 +83,46 @@ def database_backup():
         return
     ensure_backup_dir()
     database = settings.DATABASES["default"]
-    if database["ENGINE"] != "django.db.backends.postgresql":
-        return
-    cmd = ["pg_dump", "--dbname", database["NAME"]]
-    if database["HOST"]:
-        cmd += ["--host", database["HOST"]]
-    if database["PORT"]:
-        cmd += ["--port", database["PORT"]]
-    if database["USER"]:
-        cmd += ["--username", database["USER"]]
-    if settings.DATABASE_BACKUP == "compressed":
-        cmd += ["--file", data_dir("backups", "database.sql.gz")]
-        cmd += ["--compress", "6"]
+    env = get_clean_env()
+
+    if database["ENGINE"] == "django.db.backends.postgresql":
+        cmd = ["pg_dump", "--dbname", database["NAME"]]
+
+        if database["HOST"]:
+            cmd += ["--host", database["HOST"]]
+        if database["PORT"]:
+            cmd += ["--port", database["PORT"]]
+        if database["USER"]:
+            cmd += ["--username", database["USER"]]
+        if settings.DATABASE_BACKUP == "compressed":
+            cmd += ["--file", data_dir("backups", "database.sql.gz")]
+            cmd += ["--compress", "6"]
+        else:
+            cmd += ["--file", data_dir("backups", "database.sql")]
+
+        env["PGPASSWORD"] = database["PASSWORD"]
+    elif database["ENGINE"] == "django.db.backends.mysql":
+        cmd = ["mysqldump", "--databases", database["NAME"]]
```

Comment:
It should probably include `--column-statistics=0` otherwise `mysqldump` fails with older servers (which are used in the CI environment), see https://serverfault.com/q/912162

Reply:
The --column-statistics is going to be a problem as that is unsupported in MariaDB.  However, so long as the client and server are from the same supplier, it won't matter.

Reply:
But this problem exists in the CI environment. I have no clue whether it's easily possible to install matching tools there, right now we use the ones shipped in the image (what seems to be MySQL 8.0.22 right now).

Reply:
But that problem will not affect a version 8 mysqldump cli from dumping a mysql 8 database.  It would affect you attempting to dump say a MySQL 5.7-- or MariaDB 10.5-- database using a version 8 mysqldump cli.  So, there should not be a problem.

Reply:
That probably makes sense, but still we need to make it work in the CI environment as well. Can you please try rebasing on top of 45ed219f7c86fa7d905dee2678cd26a559967384 whether it helps?